.github/workflows/build.yml

```diff
@@ -92,4 +104,4 @@ jobs:
           name: nupkg-artifacts
           path: |
             ./bin/client/DotNet/G-Research.Armada.Client.${{ steps.create-release-tag.outputs.release_tag }}.nupkg
-            ./bin/client/DotNet/ArmadaProject.Io.Client.${{ steps.create-release-tag.outputs.release_tag }}.nupkg
+            ./bin/client/DotNet/ArmadaProject.Io.Client.${{ steps.create-release-tag.outputs.release_tag }}.nupkg
```

Comment:
i get what you mean, however this change comes from the formatting settings by the IDE, and i am sure you are aware of that. if we continue to skip these types of changes, not only that the formatting itself will be ignored, we will add additional work to the contributor that needs to go back to the file and remove the change. i am trying to prevent that from happening. hope that makes some sense